src/mmw/js/src/analyze/views.js

```diff
@@ -646,9 +646,9 @@ var ChartView = Marionette.ItemView.extend({
     getBarClass: function(item) {
```

Comment:
These changes seem to have had an impact on the css precedence rules of .bar-chart.nvbar::fill, resulting in the base color winning over the nlcd colors (same for soil)

![screenshot from 2017-02-20 14 48 15](https://cloud.githubusercontent.com/assets/1014341/23139715/c8341bdc-f77b-11e6-9889-530a6b57f5f0.png)